data-prepper-plugins/geoip-processor/src/test/java/org/opensearch/dataprepper/plugins/geoip/extension/MaxMindConfigTest.java

```diff
@@ -14,38 +14,41 @@
 import org.mockito.junit.jupiter.MockitoExtension;
 import org.opensearch.dataprepper.test.helper.ReflectivelySetField;
 
+import java.io.File;
 import java.net.URISyntaxException;
 import java.time.Duration;
 import java.util.Map;
+import java.util.UUID;
 
 import static org.hamcrest.CoreMatchers.equalTo;
 import static org.hamcrest.CoreMatchers.instanceOf;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.mockito.Mockito.when;
-import static org.opensearch.dataprepper.plugins.geoip.extension.MaxMindConfig.DEFAULT_DATABASE_DESTINATION;
 
 @ExtendWith(MockitoExtension.class)
 class MaxMindConfigTest {
-    private MaxMindConfig maxMindConfig;
     @Mock
     private MaxMindDatabaseConfig maxMindDatabaseConfig;
 
-    @BeforeEach
-    void setup() {
-        maxMindConfig = new MaxMindConfig();
+    private MaxMindConfig createObjectUnderTest() {
+        return new MaxMindConfig();
     }
 
-    @Test
+    @RepeatedTest(2)
```

Comment:
QUES: why are we repeating running this test case?

Reply:
Running this test twice ensures that the `System.getProperty` call is made dynamically.

This code will evaluate this once and then make it a constant:

https://github.com/opensearch-project/data-prepper/blob/67650987919f20aba06e22d284d1e66d38f42be9/data-prepper-plugins/geoip-processor/src/main/java/org/opensearch/dataprepper/plugins/geoip/extension/MaxMindConfig.java#L28

So this test passes when run once with the above code. If you run it twice, then you are sure it is getting the property dynamically.